modules/ROOT/pages/showcase-apps/device-security.adoc

```diff
@@ -1,2 +1,21 @@
 include::{partialsdir}/attributes.adoc[]
 
+= Demonstrating {device-security-service}
+
+[discrete]
+== Prerequisites
+
+* [x] You have completed all the steps in xref:showcase-apps/service-setup.adoc[Using the Showcase Apps].
```

Comment:
the reason I had 2 items linking to the same page is because we will most probably split the service setup page into 2 pages. But, let's keep it at 1 item only for now. We can always change later.